extensions-contrib/cloudfiles-extensions/src/main/java/io/druid/firehose/cloudfiles/CloudFilesBlob.java

```diff
@@ -25,41 +25,41 @@
 
 public class CloudFilesBlob
 {
-	@JsonProperty
-	@NotNull
-	private String container = null;
+  @JsonProperty
+  @NotNull
```

Comment:
Suggest doing fixes in a separate extension to reduce the clutter in this PR, also makes  collisions easier to resolve